recipes/drupal7-subfolder/README.md

```diff
@@ -0,0 +1,207 @@
+Writing a public summary of what we had to do to make DDEV work with a Drupal 7 site subfolder setup.
+
+## Subfolder
+
+A Drupal site might be operated from a subfolder (subdirectory) of a domain. Example for such a URL:
+* No Subfolder frontpage: https://example.com
+* Subfolder frontpage: https://example.com/jobs
+
+That means the Drupal site only lives in the `/jobs` part. A reverse proxy sits in front of the site and delivers requests either to the main page "/" or the Drupal site at "/jobs.
```

Comment:
You need to explain the folder structure. I assume it's a main folder called "example.com" with a subdirectory called "jobs"

But where is the ddev config done? At the top level or below? (Where does the .ddev folder live?)

Reply:
There is no file folder structure because we only want to run the /jobs subfolder site.

The ddev config is done at the usual root folder level.

Reply:
Probably best if I make a complete drupal 7 repository with the .ddev config folder in it for reference. Then you can simply clone that and run ddev start.